client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -17,95 +17,95 @@ export const MAX_POSTS_TO_SHOW = 6;
 
 function PlaceholderPostEdit( props ) {
```

Comment:
Whoa, there are 4 components defined in this file! I strongly prefer 1 component 1 file.

Reply:
They are all dependencies of a master component.